include/net/golioth/settings.h

```diff
@@ -70,6 +70,7 @@ enum golioth_settings_value_type {
 	GOLIOTH_SETTINGS_VALUE_TYPE_BOOL,
 	GOLIOTH_SETTINGS_VALUE_TYPE_FLOAT,
 	GOLIOTH_SETTINGS_VALUE_TYPE_STRING,
+	GOLIOTH_SETTINGS_VALUE_TYPE_INT64,
```

Comment:
Commit message says:
> CBOR-encoded integers only use as many bytes as
necessary, but we will treat them all as uint64_t
after decoding.

It should be `s/uint64_t/int64_t/`, right?